ante/ante.go

```diff
@@ -1,55 +1,55 @@
 package ante
 
-import (
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-	"github.com/cosmos/cosmos-sdk/x/auth/ante"
-	ibcante "github.com/cosmos/ibc-go/v3/modules/core/ante"
-	ibckeeper "github.com/cosmos/ibc-go/v3/modules/core/keeper"
-)
+// import (
+// 	sdk "github.com/cosmos/cosmos-sdk/types"
+// 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+// 	"github.com/cosmos/cosmos-sdk/x/auth/ante"
+// 	ibcante "github.com/cosmos/ibc-go/v3/modules/core/ante"
+// 	ibckeeper "github.com/cosmos/ibc-go/v3/modules/core/keeper"
+// )
 
-// HandlerOptions extend the SDK's AnteHandler options by requiring the IBC
-// channel keeper.
-type HandlerOptions struct {
-	ante.HandlerOptions
+// // HandlerOptions extend the SDK's AnteHandler options by requiring the IBC
+// // channel keeper.
+// type HandlerOptions struct {
+// 	ante.HandlerOptions
 
-	IBCkeeper            *ibckeeper.Keeper
-	BypassMinFeeMsgTypes []string
-}
+// 	IBCkeeper            *ibckeeper.Keeper
+// 	BypassMinFeeMsgTypes []string
+// }
 
-func NewAnteHandler(opts HandlerOptions) (sdk.AnteHandler, error) {
-	if opts.AccountKeeper == nil {
-		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "account keeper is required for AnteHandler")
-	}
-	if opts.BankKeeper == nil {
-		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "bank keeper is required for AnteHandler")
-	}
-	if opts.SignModeHandler == nil {
-		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "sign mode handler is required for ante builder")
-	}
+// func NewAnteHandler(opts HandlerOptions) (sdk.AnteHandler, error) {
+// 	if opts.AccountKeeper == nil {
+// 		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "account keeper is required for AnteHandler")
+// 	}
+// 	if opts.BankKeeper == nil {
+// 		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "bank keeper is required for AnteHandler")
+// 	}
+// 	if opts.SignModeHandler == nil {
+// 		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "sign mode handler is required for ante builder")
+// 	}
```

Comment:
just for later when adding back ante handler 
need to add `
	if options.IBCKeeper == nil {
		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "IBC keeper is required for middlewares")
	}
`